kolibri/core/assets/src/views/TextTruncator.vue

```diff
@@ -105,6 +110,7 @@
 <style lang="scss" scoped>
 
   .ar {
+    margin-top: 8px;
```

Comment:
Why this top-margin?

Reply:
This was from Jason Tame's PR to add more spacing between the View More/Less button and the description.